barretenberg/cpp/src/barretenberg/eccvm/eccvm_prover.cpp

```diff
@@ -98,7 +80,6 @@ void ECCVMProver::execute_log_derivative_commitments_round()
     compute_logderivative_inverse<Flavor, typename Flavor::LookupRelation>(
         prover_polynomials, relation_parameters, key->circuit_size);
     transcript->send_to_verifier(commitment_labels.lookup_inverses, commitment_key->commit(key->lookup_inverses));
-    prover_polynomials.lookup_inverses = key->lookup_inverses.share();
```

Comment:
So since these have already been shared before, this line doesn't do anything correct, as the backing_memory, size, and coeff ptr are the same?

Reply:
Correct